packages/editor/src/client/languageSwitch.tsx

```diff
@@ -32,17 +32,18 @@ export function useStickyState(
   return [value, setValue]
 }
 
+let AVAILABLE_LANG: {id: string; lang: string; name: string}[] = []
+
 export function LanguageSelector() {
-  const [uiLanguage, setUILanguage] = useStickyState(
+  AVAILABLE_LANG = [
     {id: 'en', lang: 'en_US', name: 'English'},
-    'savedValues'
-  )
+    {id: 'fr', lang: 'fr_FR', name: 'Français'},
+    {id: 'de', lang: 'de_CH', name: 'Deutsch'}
+  ]
 
-  const [, i18n] = useTranslation()
+  const [uiLanguage, setUILanguage] = useStickyState(AVAILABLE_LANG[0], 'localStorageValues')
```

Comment:
you are still saving the whole object in the localstorage. And the name is still not very descriptive.